src/target/target.c

```diff
@@ -1517,6 +1519,20 @@ int target_gdb_fileio_end(struct target *target, int retcode, int fileio_errno,
 	return target->type->gdb_fileio_end(target, retcode, fileio_errno, ctrl_c);
 }
 
+bool target_supports_gdb_query_custom(struct target *target)
```

Comment:
Will we need these 2 func after my above comment?

Reply:
Removing.  Was providing accessor functions for API delineation but not necessary.